### ISSUE TITLE
Automate downloading of indexes

### ISSUE THREAD

**Issue by lintool** [castorini/pyserini#225]
Inspired by hgf:

```python
model = T5ForConditionalGeneration.from_pretrained(model_name)
```

We should do the same for downloading of indexes:

```python
from pyserini.search import SimpleSearcher

searcher = SimpleSearcher.from_prebuilt_index('ms-marco-passage')
```

**Comment by lintool** [comment]
We already have `download_and_unpack_index`:
https://github.com/castorini/pyserini/blob/master/pyserini/util.py#L75

So this should be fairly easy...

**Comment by lintool** [comment]
PyTorch seems to keep the model files in `~/.cache/torch/transformers`... perhaps we should do the same and keep our indexes in `~/.cache/pyserini/indexes`?

**Comment by qguo96** [comment]
make a PR(#230) for this. Should we keep the gz file in our cache after extraction?

**Comment by lintool** [comment]
Closed by #230.
